src/plugins/mf_priority.cpp

```diff
@@ -805,6 +851,44 @@ static int depend_cb (flux_plugin_t *p,
 }
 
 
+/*
+ * When a job gets to the job.state.sched state, pack a timestamp to be
+ * associated with the job in the case that it is eligible to be run but
+ * cannot (e.g due to resource constraints). The timestamp will be used
+ * to calculate the "age" of a job and increase its priority according to
+ * the time it has been waiting to run.
+ */
+static int sched_cb (flux_plugin_t *p,
+                     const char *topic,
+                     flux_plugin_arg_t *args,
+                     void *data)
+{
+    double t_released = 0.0;
```

Comment:
Example here, this should be a `chrono::time_point` of whatever type you're using.